nfc/libs/NfcCoreLib/lib/NciCore/phNciNfc_DbgDescription.c

```diff
@@ -306,13 +308,15 @@ void phNciNfc_PrintNfceeModeSetCmdDescription(uint8_t *pBuff, uint16_t wLen)
 
 void phNciNfc_PrintCoreResetRspDescription(uint8_t *pBuff, uint16_t wLen)
 {
-    PHNCINFC_VALIDATE_PACKET_LENGTH(3, wLen);
+    PHNCINFC_VALIDATE_PACKET_LENGTH(1, wLen);
     PH_LOG_NCI_INFO_STR("Status: %!NCI_STATUS!", pBuff[0]);
+    PHNCINFC_VALIDATE_PACKET_LENGTH(3, wLen);
```

Comment:
I don't understand why the packet length is checked twice. Can't it be done once against '3' above instead of '1'?